pkg/component/component.go

```diff
@@ -35,14 +35,14 @@ const componentNameMaxLen = -1
 
 // Description holds all information about component
 type Description struct {
-	ComponentName      string                `json:"componentName,omitempty"`
-	ComponentImageType string                `json:"type,omitempty"`
-	Path               string                `json:"source,omitempty"`
-	URLs               []urlpkg.URL          `json:"url,omitempty"`
-	Env                []corev1.EnvVar       `json:"environment,omitempty"`
-	Storage            []storage.StorageInfo `json:"storage,omitempty"`
-	LinkedComponents   map[string][]string   `json:"linkedComponents,omitempty"`
-	LinkedServices     []string              `json:"linkedServices,omitempty"`
+	ComponentName      string
+	ComponentImageType string
```

Comment:
why did we remove the json tags? wouldn't this default to json's `snake_case` instead of `CamelCase`?

Reply:
we don't need json tags in this struct, since we have machine readable struct now

Reply:
the types.go have the tags necessary

Reply:
resolving